vtex-sites/nextjs.store
# refactor: Extract `SlideOver` component to `@faststore/ui`

## What's the purpose of this pull request?

Extract the `SlideOver` component to the `@faststore/ui` package. The component has been moved 

## How to test it?

Check that the component is still functional in the following previews:
- https://nextjs-store-storybook-git-refactor-fs-360-slide-over-faststore.vercel.app/?path=/docs/organisms-slideover--default-story
- https://preview-225--nextjs.preview.vtex.app/ (e.g. open the mini cart).

## References

- https://github.com/vtex/faststore/pull/1447

## Checklist

**Dependencies**
